mathewDavidov/assignments/slidingwindow/lc209/lc209.java

```diff
@@ -0,0 +1,52 @@
+/*
+Given an array of positive integers nums and a positive integer target, 
+return the minimal length of a contiguous subarray [numsl, numsl+1, ..., numsr-1, numsr] 
+of which the sum is greater than or equal to target. If there is no such subarray, return 0 instead.
+
+Input: target = 7, nums = [2,3,1,2,4,3]
```

Comment:
This overview is a good start but would recommend being more descriptive and detailed.